content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,474 @@
+```
+
+### Kubelet
+
+Download the [latest stable release](/releases/download/) of the Kubelet.
+
+{{< tabs name="download_kubelet" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubelet"
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+```
+-->
+
+Configure:
+
+{{< note >}}
+This tutorial uses the `10.241.1.0` network. Make sure you don't have any
+conflicts with an existing network. If necessary, use a network that is available to
+you.
+{{< /note >}}
+
+```shell
+mkdir -p /etc/kubernetes
+
+cat << EOF > /etc/kubernetes/kubelet.yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false
+authorization:
+  mode: AlwaysAllow
+enableServer: false
+logging:
+  format: text
+  sanitization: false
+readOnlyPort: 10250
+podCIDR: 10.241.1.0/24
+staticPodPath: /etc/kubernetes/manifests
+EOF
+```
+
+Install:
+
+```shell
+chmod +x kubelet
+mv kubelet /usr/bin/
+
+cat <<EOF > /etc/systemd/system/kubelet.service
+[Unit]
+Description=Kubelet
+
+[Service]
+ExecStart=/usr/bin/kubelet \
+ --config=/etc/kubernetes/kubelet.yaml
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl enable --now kubelet.service
+```
+
+Quick test:
+
+```shell
+systemctl status kubelet.service
+```
+
+Detailed service check:
+
+```shell
+journalctl -u kubelet
+```
+
+## Run a static Pod in the Kubelet
+
+Configure:
+
+```shell
+cat <<EOF > static-web.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: TCP
+EOF
+```
+
+Deploy:
+
+```shell
+mkdir -p /etc/kubernetes/manifests
+```
+
+Move the `static-web.yaml` manifest file to the `/etc/kubernetes/manifests` directory.
+
+```shell
+mv static-web.yaml /etc/kubernetes/manifests/
+```
+
+### Find out information about the Kubelet and the static Pod
+
+The cni-plugin creates a `veth` interface, retrieve the information:
+
+```shell
+ip link show | grep veth
+```
+
+The output is similar to:
+
+```
+6: veth85d4e050@if2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default
+```
+
+Use the output to retrieve the IP address of the `veth` interface:
+
+```shell
+ip addr show veth85d4e050 | grep inet
+```
+
+The output is similar to:
+
+```
+    inet 10.241.1.1/32 scope global veth85d4e050
+    inet6 fe80::f04e:1ff:fefd:9464/64 scope link
+```
+
+Using the IP address of the veth interface, query the Kubelet's API endpoint at `http://10.241.1.1:10250/pods`:
```

Comment:
Why can't we query http://localhost:10250/pods? That might be easier.

Reply:
You can, and it is easier. But finding out and using the real IP address seems more thorough (to me, at least)

Reply:
It's possibly unhelpful. We can't assume that the kubelet listens on each Pod IP address; it may well not.

Reply:
Done